crates/biome_css_formatter/src/comments.rs

```diff
@@ -90,7 +90,7 @@ impl CommentStyle for CssCommentStyle {
     ) -> CommentPlacement<Self::Language> {
         match comment.text_position() {
             CommentTextPosition::EndOfLine => {
-                handle_function_comment(comment).or_else(handle_declaration_name_comment)
+                handle_function_comment(comment).or_else(CommentPlacement::Default)
```

Comment:
I still need to study the specific implementation design of this place, but now we can see that the problem that caused this issue is here:

```ts
fn handle_declaration_name_comment(
    comment: DecoratedComment<CssLanguage>,
) -> CommentPlacement<CssLanguage> {
    match comment.preceding_node() {
        Some(following_node) if AnyCssDeclarationName::can_cast(following_node.kind()) => {
            CommentPlacement::leading(following_node.clone(), comment)
        }
        _ => CommentPlacement::Default(comment),
    }
}
```